### PR TITLE
Update api-plugin-base document

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by okkez** [comment]
I've added document about `configured_in` but I'm not sure about `configured_in` .

File: docs/v0.14/api-plugin-base.txt [hunk 1]
```diff
@@ -57,37 +60,130 @@ Base class has some methods to create configurable parameters for plugins, or to
...
+    
+    # nested sections
+    config_section :child, :param_name: "children", required: false, multi: true do
+      config_param :name, :string
+      config_param :power, :integer, default: nil
+      config_section :item, multi: true do
+        config_param :name, :string
+        config_param :flavor, :string
+      end
+    end
```
Comment (repeatedly) [inline docs/v0.14/api-plugin-base.txt]: Could you add actual configuraton for it?

```
<user>
  name xxx
  password yyy
</user>
```

and

```
<child>
  name xxx
  <item>
    name xxx
    flavior yyy
  </item>
```

Comment (okkez) [reply docs/v0.14/api-plugin-base.txt]: Sure!

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**